string  command
int32   arg1
int32   arg2
int32   arg3
int32   arg4

